jbmc/src/java_bytecode/java_bytecode_concurrency_instrumentation.cpp

```diff
@@ -354,7 +354,7 @@ static void instrument_start_thread(
 static void instrument_endThread(
```

Comment:
I am surprised to find camelCase identifiers in the code base. Maybe fix those in a separate commit.